llvm/lib/Transforms/Instrumentation/PGOCtxProfLowering.cpp

```diff
@@ -105,6 +106,12 @@ std::pair<uint32_t, uint32_t> getNumCountersAndCallsites(const Function &F) {
   }
   return {NumCounters, NumCallsites};
 }
+
+void emitUnsupportedRoot(const Function &F, StringRef Reason) {
```

Comment:
Why is this method called `emitUnsupportedRoot` and not something like `emitUnsupportedError`?

Reply:
the root is unsupported. `emitUnsupportedRootError` is probably what we want.